src/std/text.ab

```diff
@@ -1,14 +1,18 @@
-/// Replaces all occurences of a pattern in the content with the provided replace text.
+/// Replaces all occurrences of a pattern in the content with the provided replace text.
 pub fun replace(source, search, replace) {
-    return "\$\{source//{search}/{replace}}"
+    search = trust $ echo \$\{{nameof search}//\\\\/\\\\\\\\} $
+    replace = trust $ echo \$\{{nameof replace}//\\\\/\\\\\\\\} $
+    return trust $ echo \$\{{nameof source}//{search}/{replace}} $
```

Comment:
this line can be: `return "\$\{source//{search}/{replace}}"`

Reply:
also the `nameof` is useless.

Reply:
I and Ph0enixKM thought the behavior was not expected. #593 and #594 are for that. But if making two decisions in a PR is not good, I could revert here and create a new one.

Reply:
@Mte90 `nameof` in this case prevents this expression from failing in case of special symbols. And the fact that you can do `"\$my_variable"` is an unexpected behaviour. All Bash interpolations must happen in command regions `$ ... $`.